### ISSUE TITLE
can't edit games on lutris ~master

### ISSUE THREAD

**Issue by WebFreak001** [lutris/lutris#4195]
### Bug description

on current master branch (installed from AUR through lutris-git as regular lutris is currently broken due to #4141) I can start games again, but now the bottom bar is missing, right-clicking games doesn't work and an exception appears when trying to right-click.

### How to Reproduce

1. Install `lutris-git` through AUR
2. Run it
3. Try to edit a game, see that bottom bar is missing and right click doesn't work

### Expected behavior

Bottom bar should be there and right-click should work.

### Log output

```shell
Traceback (most recent call last):
  File "/usr/lib/python3.10/site-packages/lutris/gui/views/base.py", line 52, in popup_contextual_menu
    self.contextual_menu.popup(event, game_actions)
  File "/usr/lib/python3.10/site-packages/lutris/gui/widgets/contextual_menu.py", line 53, in popup
    displayed = game_actions.get_displayed_entries()
  File "/usr/lib/python3.10/site-packages/lutris/game_actions.py", line 141, in get_displayed_entries
    and not steam_shortcut.all_shortcuts_set(self.game)
  File "/usr/lib/python3.10/site-packages/lutris/util/steam/shortcut.py", line 48, in all_shortcuts_set
    shortcut_found = [
  File "/usr/lib/python3.10/site-packages/lutris/util/steam/shortcut.py", line 50, in <listcomp>
    if game.name in s['AppName']
KeyError: 'AppName'
```
```


### System Information

```shell
OS:              Arch Linux rolling n/a
Arch:            x86_64
Kernel:          5.17.1-zen1-1-zen
Desktop:         i3
Display Server:  x11
```

### Checklist:

- [X] I'm not asking for support with a game or the wine runner.
- [X] I have followed the above mentioned guides and have all the graphics and wine dependencies installed.
- [X] I have checked for existing issues that describe my problem prior to opening this one.
- [X] I understand that improperly formatted bug reports may be closed without explanation.

**Comment by danieljohnson2** [comment]
I have a theory about this, but no Steam account to test with. And anyway, this is probably specific to your shortcuts.vdf file.

I read [here](https://github.com/SteamGridDB/steam-rom-manager/issues/2) that somehow Steam writes out the shortcuts with 'AppName' in lower-case. Lutris will fail if that happens.

I have a branch that tries to adjust the Lutris code to tolerate this. It's my [Appname_or_appname](https://github.com/danieljohnson2/lutris/tree/dj/AppName_or_appname) branch.

Would it be possible for you to grab this branch and try it to see if it fixes things for you? I'll pop a PR if it appears to work, but I can't test it.

**Comment by Crono141** [comment]
I also have this problem.  #4196

**Comment by Crono141** [comment]
> I have a theory about this, but no Steam account to test with. And anyway, this is probably specific to your shortcuts.vdf file.
> 
> I read [here](https://github.com/SteamGridDB/steam-rom-manager/issues/2) that somehow Steam writes out the shortcuts with 'AppName' in lower-case. Lutris will fail if that happens.
> 
> I have a branch that tries to adjust the Lutris code to tolerate this. It's my [Appname_or_appname](https://github.com/danieljohnson2/lutris/tree/dj/AppName_or_appname) branch.
> 
> Would it be possible for you to grab this branch and try it to see if it fixes things for you? I'll pop a PR if it appears to work, but I can't test it.

This build doesn't even launch.  Here's the traceback

>     ~/Downloads/lutris    master  lutris                                                                                                     ✔  8s  
> Traceback (most recent call last):
>   File "/usr/bin/lutris", line 4, in <module>
>     __import__('pkg_resources').run_script('lutris==0.5.10', 'lutris')
>   File "/usr/lib/python3.10/site-packages/pkg_resources/__init__.py", line 656, in run_script
>     self.require(requires)[0].run_script(script_name, ns)
>   File "/usr/lib/python3.10/site-packages/pkg_resources/__init__.py", line 1453, in run_script
>     exec(code, namespace, namespace)
>   File "/usr/lib/python3.10/site-packages/lutris-0.5.10-py3.10.egg/EGG-INFO/scripts/lutris", line 52, in <module>
>     from lutris.gui.application import Application  # pylint: disable=no-name-in-module
>   File "/usr/lib/python3.10/site-packages/lutris-0.5.10-py3.10.egg/lutris/gui/application.py", line 47, in <module>
>     from lutris.startup import init_lutris, run_all_checks, update_runtime
>   File "/usr/lib/python3.10/site-packages/lutris-0.5.10-py3.10.egg/lutris/startup.py", line 12, in <module>
>     from lutris.runners.json import load_json_runners
>   File "/usr/lib/python3.10/site-packages/lutris-0.5.10-py3.10.egg/lutris/runners/json.py", line 10, in <module>
>     os.path.join(datapath.get(), "json"),
>   File "/usr/lib/python3.10/site-packages/lutris-0.5.10-py3.10.egg/lutris/util/datapath.py", line 27, in get
>     raise IOError("data_path can't be found at : %s" % data_path)
> OSError: data_path can't be found at : /usr/share/lutris
>

**Comment by WebFreak001** [comment]
> I have a theory about this, but no Steam account to test with. And anyway, this is probably specific to your shortcuts.vdf file.
> 
> I read [here](https://github.com/SteamGridDB/steam-rom-manager/issues/2) that somehow Steam writes out the shortcuts with 'AppName' in lower-case. Lutris will fail if that happens.
> 
> I have a branch that tries to adjust the Lutris code to tolerate this. It's my [Appname_or_appname](https://github.com/danieljohnson2/lutris/tree/dj/AppName_or_appname?rgh-link-date=2022-04-05T22%3A26%3A12Z) branch.
> 
> Would it be possible for you to grab this branch and try it to see if it fixes things for you? I'll pop a PR if it appears to work, but I can't test it.

yes this fixed the issue for me, I cloned lutris master manually first which ran and had my problem, then tried your branch which ran and resolved the problem.

At first I thought this wouldn't be related because the game I was trying to edit wasn't a steam game, but seems like it fixed it anyway.

**Comment by HachebeDLC** [comment]
+1 on @danieljohnson2 change, applied it to the shortcut.py file from the flatpak and the context menu now works, you beat me to it!

**Comment by danieljohnson2** [comment]
Oh dear. I'm not sure what you've installed exactly, but you should probably uninstall and reinstall Lutris via your package manger; that's what worked for me the last time I had to troubleshoot such a problem with it.

The code is running  because it has found a 'shortcuts.vdf' file. If you do not use Steam, perhaps you could find and remove that file. Lutris looks for it in several directories, mostly hidden ones:
* ~/.steam
* ~/.local/share/steam
* ~/.local/share/Steam
* ~/.steam/steam
* ~/.var/app/com.valvesoftware.Steam/data/steam
* ~/.steam/debian-installation
* /usr/share/steam
* /usr/local/share/steam

I can see from your new traceback that you are *not* running the new branch; the line of code it references is not there anymore.

The way to run the latest master is like this:

git clone https://github.com/lutris/lutris.git
cd lutris
./bin/lutris -d

Note that the last command is not "lutris"; the initial dot makes the difference. Or that's my guess as it what's gone wrong.

**Comment by danieljohnson2** [comment]
That's enough testimonials for me; there's the PR with this fix.

Though it's not clear that this fix makes the Steam shortcuts *work* if you use Steam and have this problem, instead of just not crashing. But I'll take not-crashing.

**Comment by tg137** [comment]
+1 for the shortcuts.py fix, thanks @danieljohnson2 🎉

**Comment by misieck** [comment]
It happens on my Ubuntu 21.10 with GOG games, so not Steam-specific.

**Comment by danieljohnson2** [comment]
It can happen to GOG games if you have a naughty shortcuts.vdf file that it can find; the code checks to see if steam knows the game, but crashes in the attempt (though it is not a steam game). I'd guess that you have Steam installed, though the game you are testing with is from GOG.

If your problem is this one, the 'master' branch should fix it. Can you give that a try?

**Comment by misieck** [comment]
Yes I do have steam installed and yes master from git does solve that particular issue.

**Comment by strycore** [comment]
We'll ship a fix soon, in 0.5.10.1
